Authentication/SignUp/ViewModel/SignupCredentialsValidator.swift

```diff
@@ -24,7 +24,7 @@ public struct SignupCredentialsValidator {
         ])
     }
     
-    public static func defaultPasswordValidator(minLength: Int = 4, maxLength: Int? = 30) -> TextInputValidatorType {
+    public static func defaultPasswordValidator(_ minLength: Int = 4, maxLength: Int? = 30) -> TextInputValidatorType {
```

Comment:
Or only `minLength`? Because as it is an optional parameter it would be weird to being able to call this functions these ways:
```
defaultPasswordValidator(withMinLength: 5)
defaultPasswordValidator(maxLength: 5)

```
I would add no `with` anywhere, but ask for the `minLength` label.